packages/babel-helpers/src/helpers/toPrimitive.js

```diff
@@ -0,0 +1,16 @@
+/* @minVersion 7.1.5 */
+
+// https://tc39.es/ecma262/#sec-toprimitive
+export default function _toPrimitive(
```

Comment:
Can we inline this in the toPropertyKey helper?

Reply:
This helper has been exposed since 7.1.5, so probably not in the patch release. Here I extracted `toPrimitive` and `toPropertyKey` into single files.